deploy.sh

```diff
@@ -138,10 +138,12 @@ function modifyTaskDefinitionFile() {
             exit 1;
         fi
 
-        touch tmpfile;
+        if [[ "$IMAGE" != "NO_IMAGE" ]]; then
```

Comment:
Can we put a little comment in this above this line to remember why we did this? You don't have to name drop clients/things, just a generic reason why.

Reply:
Addded comment